app/models/fullcalendar_engine/event_series.rb

```diff
@@ -1,7 +1,7 @@
 module FullcalendarEngine
   class EventSeries < ActiveRecord::Base
     
-    attr_accessor :title, :description, :commit_button
+    attr_accessor :title, :description, :commit_button, :color
```

Comment:
I am saying that you don't need to put it in the accessor list. It will always be accessible.